src/test/resources/META-INF/mods.toml

```diff
@@ -240,7 +240,8 @@ modId="custom_transformtype_test"
 modId="mega_model_test"
 [[mods]]
 modId="emissive_elements_test"
-
```

Comment:
Please keep the blank line.

Reply:
So that there's 3 blank line between `# ADD ABOVE THIS LINE` and `modId="item_use_animation_test"`?

Reply:
Yep, so there are only additions to this file.

Reply:
Done